pkg/controller/stacks/install/stackinstall.go

```diff
@@ -252,17 +252,18 @@ func (h *stackInstallHandler) update(ctx context.Context) (reconcile.Result, err
 
 func (h *stackInstallHandler) delete(ctx context.Context) (reconcile.Result, error) {
 	labels := map[string]string{
-		labelInstallNamespace: h.ext.GetNamespace(),
-		labelInstallName:      h.ext.GetName(),
-		labelInstallUID:       string(h.ext.GetUID()),
+		labelParentKind:      h.ext.GetObjectKind().GroupVersionKind().Kind,
+		labelParentNamespace: h.ext.GetNamespace(),
+		labelParentName:      h.ext.GetName(),
+		labelParentUID:       string(h.ext.GetUID()),
 	}
 	list := &apiextensionsv1beta1.CustomResourceDefinitionList{}
 	if err := h.kube.List(ctx, list, client.MatchingLabels(labels)); err != nil {
 		return fail(ctx, h.kube, h.ext, err)
 	}
 
 	for i := range list.Items {
-		if err := h.kube.Delete(ctx, &list.Items[i]); err != nil {
+		if err := h.kube.Delete(ctx, &list.Items[i]); err != nil && !kerrors.IsNotFound(err) {
```

Comment:
You could also use `if err := resource.IgnoreNotFound(h.kube.Delete(ctx, &list.Items[i])); err != nil` here, though neither is significantly more succinct or readable.

Reply:
I go for `if err := h.kube.Delete(ctx, &list.Items[i]); resource.IgnoreNotFound(err) != nil` for readability.